qa/pom.xml

```
@@ -55,7 +55,7 @@
     <dependency>
       <groupId>io.camunda</groupId>
       <artifactId>camunda-process-test-spring</artifactId>
-      <version>${version.camunda-8}</version>
+      <version>8.9.0-SNAPSHOT</version>
```

When CPT 8.8 add support
```suggestion
      <version>${version.camunda-8}</version>
```